bin/stellar-core-backup

```diff
@@ -4,35 +4,51 @@ require 'stellar-core-backup'
 require 'optparse'
 
 
-options = { verify: 'V'}
+options = { listlen: 1, verify: true}
 
-OptionParser.new do |opts|
-  opts.banner = "Usage: #{$0} [options]"
-  opts.on("-b", "--backup", "cold backup stellar-core") do |backup|
-    options[:type] = 'backup'
-  end
-  opts.on("-c CONFIG", "--config", "configuration file to use") do |config|
-    options[:config] = config
-  end
-  opts.on("-C", "--clean", "auto-clean buckets directory") do |clean|
-    options[:clean] = 'clean'
-  end
-  opts.on("-g", "--getkey", "load or update the configured public GPG key") do |getkey|
-    options[:type] = 'getkey'
-  end
-  opts.on("-l", "--list", "list all backups") do |list|
-    options[:list] = 'list'
-  end
-  opts.on("-r", "--restore", "offline restore, uses latest backup") do |restore|
-    options[:type] = 'restore'
-  end
-  opts.on("--no-verify", "don't PGP sign backed up shasum file or verify restore files integrity (default on)") do |notverify|
-    options.delete(:verify)
-  end
-end.parse!
+begin
+  OptionParser.new do |opts|
+    opts.banner = "Usage: #{$0} [options]"
```

Comment:
I find adding this much logic inside the optparse block confusing, it also has the potential of becoming difficult to maintain.

If you we have mutually exclusive options that we want to enforce this should be done outside of the block imo

Reply:
If we choose to enforce this I'll come up with a more effective way.